docs/usage.md

```diff
@@ -14,22 +14,22 @@ Describe the next run using Markdown
 $ wandb describe
 ```
 
-Checkout the commit associated with the bucket and restore the config variables
+Checkout the commit associated with the run and restore the config variables
 
 ```console
-$ wandb restore
+$ wandb restore run-id
```

Comment:
I was thinking, alongside restore we could potentially have `wandb rerun run-id` which would restore and run the same command as used before.  Was also thinking we probably don't want to download the non-wandb run files when restoring...